### PR TITLE
Make README.md build instructions more specific

### PR DESCRIPTION
**Pull request by designsimply**
* Make the command line steps in the build instructions a little more specific.
* Move the `OAuth2 Authentication` up to be closer to the build instructions.

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,26 +9,61 @@ it on [Google Play](https://play.google.com/store/apps/details?id=org.wordpress.
...
 
 1. Make sure you've installed [JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) and [Android Studio](https://developer.android.com/studio/index.html), a _Standard Setup_ would work.
-2. Clone this GitHub repository.
-3. Copy `gradle.properties-example` to `gradle.properties`.
-4. In Android Studio open the project from the local repository as a **Gradle project** (this will auto-generate `local.properties` with the SDK location).
-5. Make sure you have an emulation device setup in AVD Manager (_Tools → Android → AVD Manager_).
-6. Run.
+1. `git clone git@github.com:wordpress-mobile/WordPress-Android.git` in the folder of your preference.
+1. `cd WordPress-Android` to enter the working directory.
+1. `cp gradle.properties-example gradle.properties` to copy the example credentials file to your home folder.
```
Comment (jkmassel) [inline README.md]: This command won't copy anything to the home folder 🤔 

Perhaps:

>to set up the sample app credentials file.

Comment (designsimply) [reply README.md]: Aha 🤔 yes, thanks for catching that!

File: README.md [hunk 2]
```diff
@@ -9,26 +9,61 @@ it on [Google Play](https://play.google.com/store/apps/details?id=org.wordpress.
...
-2. Clone this GitHub repository.
-3. Copy `gradle.properties-example` to `gradle.properties`.
-4. In Android Studio open the project from the local repository as a **Gradle project** (this will auto-generate `local.properties` with the SDK location).
-5. Make sure you have an emulation device setup in AVD Manager (_Tools → Android → AVD Manager_).
-6. Run.
+1. `git clone git@github.com:wordpress-mobile/WordPress-Android.git` in the folder of your preference.
+1. `cd WordPress-Android` to enter the working directory.
+1. `cp gradle.properties-example gradle.properties` to set up the sample app credentials file.
+1. In Android Studio, open the project from the local repository as a **Gradle project** to auto-generate `local.properties` with the SDK location.
+1. Go Tools → Android → AVD Manager and make sure you have an emulation device setup.
```
Comment (JavonDavis) [inline README.md]:
```suggestion
1. Go to Tools → Android → AVD Manager and make sure you have an emulation device setup.
```

Comment (designsimply) [reply README.md]: Good catch. Thank you!

**Comment by loremattei** [comment]
Hey! I'm moving this to 12.6 as 12.5 has been cut.

**Pull request merged**